templates/newsletter/form.html.twig

```diff
@@ -0,0 +1,5 @@
+<form method="post" action="{{ path('newsletter_subscription') }}" name="app_newsletter_banner">
```

Comment:
Unless for a very good reason, this part should be rendered using `form` function

Reply:
totaly right, I was just used what it was done in previous newsletter forms rendered